code/modules/station_goals/bluespace_tap.dm

```diff
@@ -376,10 +318,38 @@
 	consume_direct_power(mining_power + stabilizer_power)
 
 	// 2 points per 50 KW up to 20 and 4 points per MW (or 5 when emmaged).
-	mined_points = min(BASE_POINTS * (mining_power / (50 KW)) , 20) + mining_power * (POINTS_PER_W + emagged / (1 MW))
-	points += mined_points
-	total_points += mined_points
-	update_icon()
+	// A dirty machine does not produce points
+	if(!dirty)
+		mined_points = min(BASE_POINTS * (mining_power / (50 KW)) , 20) + mining_power * (POINTS_PER_W + emagged / (1 MW))
+		points += mined_points
+		total_points += mined_points
+		update_icon()
+
+	// Handle automatic spawning of loot. Extra loot spawns in maints if stabilizers are off to incentivise risk taking
+	if(total_points > clothing_interval)
+		produce(product_list[1], FALSE, !stabilizers)
+		radio.autosay("<b>Bluespace harvester progress detected: [src] has produced some clothes!</b>", name, "Engineering")
```

Comment:
It sound very "abstract" for lack of a better term to use some here, something like this might be better for these autosays
```suggestion
		radio.autosay("<b>Bluespace harvester progress detected: [src] has produced unknown clothing!</b>", name, "Engineering")
```